ontology/yaml/resources/METERS/entity_types/HM.yaml

```diff
@@ -48,3 +48,23 @@ HM_HW:
   - heating_return_water_temperature_sensor
   - heating_thermal_power_sensor
   - heating_thermal_energy_accumulator
+
+HM_HW_WAC:
+  description: "Heating load heat meter with total heating water volume accumulator."
+  implements:
+  - HM_HW
+  is_canonical: true
+  uses:
+  - water_volume_accumulator
+
+HM_GTW:
```

Comment:
since this is clear it applies to the geothermal loop (because the meter will have a name like `HM-1_GTWL` or something) is it necessary to include the geothermal qualifier on this device? it is not distinguishing it from anything else on that particular device.